lib/audit/catalog_to_moab.rb

```diff
@@ -59,6 +59,10 @@ def self.check_version_all_dirs_profiled(last_checked_b4_date)
 
     # TODO: anything special if preserved_copy.status is not OK_STATUS? - see #480
 
+    unless preserved_copy.matches_po_current_version?
+      results.add_result(PreservedObjectHandlerResults::PC_PO_VERSION_MISMATCH, { pc_version: preserved_copy.version, po_version: preserved_copy.preserved_object.current_version })
```

Comment:
looking at the analogous code in M2C, this should stop processing, so needs a "return" line.